guides/common/modules/proc_tuning-apache-httpd-child-processes.adoc

```diff
@@ -0,0 +1,27 @@
+[id="tuning_apache_httpd_child_processes_{context}"]
+= Tuning Apache httpd child processes
+
+By default, HTTPD uses event request handling mechanism.
+When the number of requests to httpd exceeds the maximum number of child processes that can be launched to handle the incoming connections, httpd raises an HTTP 503 Service Unavailable error.
+Amidst httpd running out of processes to handle, the incoming connections can also result in multiple component failures on your {Project} services side due to the dependency of components like Pulp on the availability of httpd processes.
+
+You can adapt the configuration of HTTPD event to handle more concurrent requests based on your expected peak load.
+
+.Procedure
+. Modify the number of concurrent requests in '/etc/foreman-installer/custom-hiera.yaml' by changing or adding the following lines:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+apache::mod::event::serverlimit: 64
+apache::mod::event::maxrequestworkers: 1024
+apache::mod::event::maxrequestsperchild: 4000
+----
+. Apply your changes to {ProjectServer}.
+For more information, see xref:Applying_configurations_{context}[].
+
+The example is identical to running `{foreman-installer} --tuning=medium` on {ProjectServer}.
+
+[WARNING]
```

Comment:
This warning should be above the procedure IMHO. Readers should be aware that they're making their installation essentially unsupported before they start, not after.

Another caveat is that tuning is Katello-only, so it should be hidden behind `ifdef::katello,orcharhino,satellite[]`.

Reply:
Yes, I do agree on this point. Change has been done.